c7n/mu.py

```diff
@@ -991,6 +991,86 @@ def remove(self, func):
                     raise
 
 
+class SimpleNotificationServiceSubscription(object):
```

Comment:
Done in f0ffe5de4927f68dd9a021e36715ebca97f0caeb.

Ftr I was patterning on `CloudWatchLogSubscription`. 😜